examples/spark/spark-gluster/spark-master-controller.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: spark-master
-          image: gcr.io/google_containers/spark-master:1.5.1_v2
+          image: gcr.io/google_containers/spark-master:1.5.2_v1
```

Comment:
this needs to be changed to spark:1.5.2_v1 and the command: /start-master needs to be added